lib/src/lib/table/components/table/table.component.ts

```diff
@@ -205,11 +209,17 @@ export class Lab900TableComponent implements OnChanges {
 
   public displayedColumns: string[] = [];
 
+  public showCellFooters = false;
+
   // when columnOrder is not specified, put them in the back (position 10000)
   public static reorderColumnsFn(a: TableCell, b: TableCell): number {
     return (a.columnOrder ?? 10000) - (b.columnOrder ?? 10000);
   }
 
+  public ngAfterContentInit(): void {
+    this.showCellFooters = !!this.tableCells.find((cell) => cell.footer);
```

Comment:
you can use `yourArray.some()`, this returns a boolean if it finds a matching element

Reply:
done